{
  "contest": 142,
  "handle": "IllIllI",
  "address": "0xCC6B0a35481EA40b60780B7c0294c6c29061038f",
  "risk": "2",
  "title": "Unbounded loops may cause `exercise()`s and `withdraw()`s to fail ",
  "issueId": 227,
  "issueUrl": "https://github.com/code-423n4/2022-06-putty-findings/issues/227"
}